dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jaxwscdi-2.3.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.jaxwscdi-2.3
+visibility=private
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.jaxws-2.3))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.cdi-1.2)(osgi.identity=com.ibm.websphere.appserver.cdi-2.0)))"
```

Comment:
remove cdi-1.2 as a feature for auto provisioning since ideally this will be a javaee-8.0 feature